crates/unleash-edge-enterprise/src/lib.rs

```diff
@@ -1,15 +1,17 @@
 use std::{pin::Pin, sync::Arc};
 use tokio::sync::watch::Sender;
 use tracing::{debug, warn};
+use ulid::Ulid;
 use unleash_edge_http_client::UnleashClient;
 use unleash_edge_types::{RefreshState, enterprise::LicenseStateResponse, tokens::EdgeToken};
 
 async fn send_heartbeat(
```

Comment:
Sending heartbeats now requires a connection id so we can send the downstream instance's connection id. Probably not worth it for this to have a whole different method from our existing heartbeat method